docs/_docs/index.md

```diff
@@ -4,53 +4,44 @@ permalink: /docs/home/
 redirect_from: /docs/index.html
 ---
 
-This site aims to be a comprehensive guide to Jekyll. We’ll cover topics such
-as getting your site up and running, creating and managing your content,
-customizing the way your site works and looks, deploying to various
-environments, and give you some advice on participating in the future
-development of Jekyll itself.
-
-## So what is Jekyll, exactly?
-
-Jekyll is a simple, blog-aware, static site generator. It takes a template
-directory containing raw text files in various formats, runs it through
-a converter (like [Markdown](https://daringfireball.net/projects/markdown/))
-and our [Liquid](https://shopify.github.io/liquid/) renderer, and
-spits out a complete, ready-to-publish static website suitable
-for serving with your favorite web server. Jekyll also happens to be the engine
```

Comment:
I feel like this explanation makes total sense to someone who uses Jekyll, but to a newcomer it might feel dense. There's a lot of moving parts, with no room between the commas for the weight of each part to sink in. Thoughts?